config/ember-try.js

```diff
@@ -9,19 +9,32 @@ module.exports = async function () {
         npm: {
           devDependencies: {
             'ember-source': '~3.28.0',
+            'ember-qunit': '^5.0.0',
+            '@ember/test-helpers': '^2.4.0',
           },
         },
       },
       {
-        name: 'ember-default-with-jquery',
```

Comment:
This PR drops this scenario. There's no real need to test this addon with jquery-integration enabled, because it doesn't use jquery.